standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -344,6 +346,7 @@ public void markCleaned(CompactionInfo info) throws MetaException {
         rs = pStmt.executeQuery();
         if (rs.next()) {
           info = CompactionInfo.loadFullFromCompactionQueue(rs);
```

Comment:
This code is quit weird. We should not read back the CompactionInfo from the db, we could just use insert select for the COMPLETED_COMPACTIONS. The other used fields later are already in the info passed by the Cleaner

Reply:
refactored

Reply:
Looks better, thanks